src/Diagnostics.DataProviders/DataProviders.cs

```diff
@@ -29,7 +29,9 @@ public DataProviders(DataProviderContext context)
             Kusto = GetOrAddDataProvider(new KustoLogDecorator(context, new KustoDataProvider(_cache,
                      context.Configuration.KustoConfiguration,
                      context.RequestId,
-                     context.KustoHeartBeatService)));
+                     context.KustoHeartBeatService,
+                     context.QueryStartTime, 
```

Comment:
Where is the value for `context.QueryStartTime` & `context.QueryEndTime` getting set?

Reply:
Looks like they were set in the `BeginRequestHandle` function of `DiagnosticsRequestMiddleware.cs`. I've verified the timestamps were correct by step debugging and checking the query in kusto
![image](https://user-images.githubusercontent.com/68874830/131417643-b3b6ac8c-49c2-4d73-9b64-fd85b61b52b6.png)